pyt/fixed_point.py

```diff
@@ -6,10 +6,10 @@ class FixedPointAnalysis():
     """Run the fix point analysis."""
 
     def __init__(self, cfg, analysis):
-        """Fixed point analysis
+        """Fixed point analysis.
 
-        analysis must be a dataflow analysis containing a 'fixpointmethod'
-        method that analyzes one CFG node"""
+        Analysis must be a dataflow analysis containing a 'fixpointmethod'
+        method that analyzes one CFG node."""
```

Comment:
probably should fix "analyzes" here. It is with a z here and we have used british english in most of the project i am pretty sure :)